tests/components/automation/test_state.py

```diff
@@ -950,19 +950,19 @@ async def test_if_fires_on_entities_change_overlap_for_template(hass, calls):
         async_fire_time_changed(hass, mock_utcnow.return_value)
         hass.states.async_set("test.entity_2", "world")
         await hass.async_block_till_done()
-        assert 0 == len(calls)
+        assert len(calls) == 0
         mock_utcnow.return_value += timedelta(seconds=3)
         async_fire_time_changed(hass, mock_utcnow.return_value)
         await hass.async_block_till_done()
-        assert 1 == len(calls)
-        assert "test.entity_1 - 0:00:05" == calls[0].data["some"]
+        assert len(calls) == 1
+        assert calls[0].data["some"] == "test.entity_1 - 0:00:05"
 
         mock_utcnow.return_value += timedelta(seconds=3)
         async_fire_time_changed(hass, mock_utcnow.return_value)
         await hass.async_block_till_done()
-        assert 1 == len(calls)
+        assert len(calls) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5456697)